src/components/ApplicationForm/Questionnaire.js

```diff
@@ -188,6 +188,34 @@ export default ({ errors, formInputs, onChange }) => {
             })
           }
         />
+        <Select
+          type="checkbox"
+          label="cmd-f 2022"
+          checked={formInputs.eventsAttended.cmdf2022}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              eventsAttended: {
+                ...formInputs.eventsAttended,
+                cmdf2022: !formInputs.eventsAttended.cmdf2022,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="cmd-f 2023"
+          checked={formInputs.eventsAttended.cmdf2023}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              eventsAttended: {
+                ...formInputs.eventsAttended,
+                cmdf2023: !formInputs.eventsAttended.cmdf2023,
```

Comment:
label should be different (as well as line 208 `{formInputs.eventsAttended.cmdf2023}`)